eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -351,4 +351,22 @@ function Remove-EmptySections {
     }
     $ChangeLogEntry.ReleaseContent = $sanitizedReleaseContent.ToArray()
   }
+}
+
+function  Get-LatestReleaseDateFromChangeLog
+{
+  param (
+    [Parameter(Mandatory = $true)]
+    $ChangeLogLocation
+  )
+  $changeLogEntries = Get-ChangeLogEntries -ChangeLogLocation $ChangeLogLocation
+  $latestVersion = $changeLogEntries[0].ReleaseStatus.Trim("()")
+  if ($latestVersion -as [DateTime])
+  {
+    return $latestVersion
+  }
+  else
+  {
+    return $null
+  }
```

Comment:
```suggestion
  return ($latestVersion -as [DateTime])
```
as will return null if the type doesn't match, but if the type does match you should return the DateTime object which this does as well.